packages/webpack/__e2e-fixtures__/typescript/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const webpack2 = require('../../index')
+
+// Need to write it like this instead of destructuring so it runs on Node 4.x w/o transpiling
+const addPlugins = webpack2.addPlugins
+const createConfig = webpack2.createConfig
+const entryPoint = webpack2.entryPoint
+const performance = webpack2.performance
+const setOutput = webpack2.setOutput
+const webpack = webpack2.webpack
+
+const typescript = require('@webpack-blocks/typescript')
+const path = require('path')
+
+module.exports = createConfig([
+  entryPoint(
+    path.join(__dirname, 'app.ts')
+  ),
+  setOutput(
+    path.join(__dirname, 'build/bundle.js')
+  ),
+  typescript(),
+  performance({
+    maxAssetSize: 100000,
+    maxEntrypointSize: 500000,
+    hints: 'error'
+  }),
+  addPlugins([
+    new webpack.DefinePlugin({
```

Comment:
I'd say we replace that by `defineConstants()`. But since this is already tested by the babel/postcss/extract-text e2e test I would just throw out the `process.env` injection thing altogether 😉